relayer/processor/path_processor.go

```diff
@@ -354,3 +380,114 @@ func (pp *PathProcessor) Run(ctx context.Context, cancel func()) {
 		}
 	}
 }
+
+func (pp *PathProcessor) handleLocalhostData(cacheData ChainProcessorCacheData) {
+	pathEnd1Cache := ChainProcessorCacheData{
+		IBCMessagesCache:     NewIBCMessagesCache(),
+		InSync:               cacheData.InSync,
+		ClientState:          cacheData.ClientState,
+		ConnectionStateCache: cacheData.ConnectionStateCache,
+		ChannelStateCache:    cacheData.ChannelStateCache,
+		LatestBlock:          cacheData.LatestBlock,
+		LatestHeader:         cacheData.LatestHeader,
+		IBCHeaderCache:       cacheData.IBCHeaderCache,
+	}
+	pathEnd2Cache := ChainProcessorCacheData{
+		IBCMessagesCache:     NewIBCMessagesCache(),
+		InSync:               cacheData.InSync,
+		ClientState:          cacheData.ClientState,
+		ConnectionStateCache: cacheData.ConnectionStateCache,
+		ChannelStateCache:    cacheData.ChannelStateCache,
+		LatestBlock:          cacheData.LatestBlock,
+		LatestHeader:         cacheData.LatestHeader,
+		IBCHeaderCache:       cacheData.IBCHeaderCache,
+	}
+
+	// split up data and send lower channel-id data to pathEnd1 and higher channel-id data to pathEnd2
+	for k, v := range cacheData.IBCMessagesCache.PacketFlow {
+		chan1, err := chantypes.ParseChannelSequence(k.ChannelID)
+		if err != nil {
+			pp.log.Error("failed to parse channel ID int from string", zap.Error(err))
```

Comment:
I think this and the subsequent log should start with an uppercase letter for consistency with other log messages.